pyodide_build/mkpkg.py

```diff
@@ -123,6 +169,11 @@ def make_parser(parser):
 complex things.""".strip()
     parser.add_argument("package", type=str, nargs=1, help="The package name on PyPI")
     parser.add_argument("--update", action="store_true", help="Update existing package")
+    parser.add_argument(
+        "--update-if-not-patched",
```

Comment:
Maybe rather `--update-if-patched` (i.e. don't update patched packages by default) which is a safer default choice and matches the current defaults.

Reply:
I think when using `mkpkg` to update just one package it's better if it just updates and prints the warning? If you do `./bin/pyodide mkpkg update whatever_package` and then can't figure out how to migrate the patches you can always `git checkout packages/whatever_package` to restore it. It's really just when updating in bulk that it's a nuisance if it updates the patched ones.

Reply:
Agreed, I haven't done that in a while and forgot about that use-case. Lately I was mostly using `update-all`.